knative-operator/pkg/webhook/webhook.go

```diff
@@ -31,26 +27,12 @@ func AddToManager(m manager.Manager) error {
 	}
 
 	log.Info("Setting up webhook server")
-	operatorNs := os.Getenv(common.NamespaceEnvKey)
-	if operatorNs == "" {
-		return errors.New("NAMESPACE not provided via environment")
-	}
 	// This will be started when the Manager is started
+	truePtr := true
```

Comment:
you can use:
```
import "knative.dev/pkg/ptr"
DisableWebhookConfigInstaller: ptr.Bool(true),
```

Reply:
I wanted to avoid pulling this package in, but we indeed already have it somewhere else. Adding.